test/extended/builds/pipeline.go

```diff
@@ -87,10 +89,23 @@ var _ = g.Describe("[Feature:Builds][Slow] openshift pipeline build", func() {
 			// an override to use that images stream and test image
 			var licensePrefix, pluginName string
 			useSnapshotImage := false
+
+			// create persistent volumes if running persistent jenkins
+			if jenkinsTemplatePath == jenkinsPersistentTemplatePath {
+				_, err := exutil.SetupHostPathVolumes(
```

Comment:
we've gotten beat up about our use of this before because it only works on a single node cluster when the test code is running on the master, which means this test can never be part of our conformance bucket.

Can you find a way to do this w/o persistent volumes?  Or do we need to talk to the CI team about ensuring our test clusters have PVs available?